code/modules/research/techweb/all_nodes.dm

```diff
@@ -1065,6 +1065,18 @@
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 
+/datum/techweb_node/ai_basic/New()
+	. = ..()
+	if(HAS_TRAIT(SSstation, STATION_TRAIT_HUMAN_AI))
```

Comment:
Oh, now I'm a little conflicted about the previous review comments suggesting to add `is_station_level()` checks to several statements dictating whether the item should init or die...

Reply:
The report text I gave it is that there was some hinderance in AI development so in this 'timeline', the technology for making AIs doesn't exist yet, so I wanted it to affect off-station too.

Reply:
Ok, got it.